bottles/frontend/views/bottle_dependencies.py

```diff
@@ -113,12 +113,11 @@ def new_dependency(dependency, plain=False):
             self.__registry.append(entry)
             self.list_dependencies.append(entry)
 
-        def callback(result, error=False):
+        def callback(_result, _error=False):
             self.stack.set_visible_child_name("page_deps")
 
         def process_dependencies():
             time.sleep(.3)  # workaround for freezing bug on bottle load
-            RepoStatus.repo_wait_operation("dependencies.fetching")
             RepoStatus.repo_wait_operation("dependencies.organizing")
```

Comment:
should this be removed instead of removing wait for fetching?

Reply:
No, here we want to have the fetching and organizing done. As the organizing already depends on the fetching, it's not necessary. This was a safe-guard because of the old way I used locks that could create problems.